### ISSUE TITLE
confusing phonetic transcription

### ISSUE THREAD

**Issue by chanagaray** [getify/TNG-Hooks#25]
I'm supposing hooks is pronounced /hʊks/ as usual but now I'm not certain how to pronounce TNG, is that notation supposed to be IPA or is it some more obscure standard?

**Comment by getify** [comment]
Since it's a made up sort of combined word name, I think I just used google:

https://www.google.com/search?q=phonetic+pronunciation+ting

As you can see, google's output there, for the american english pronounciation, just literally uses "ting". That seems rather... boring/plain to me, but I didn't want to overthink google.

Or here's another link: https://www.howtopronounce.com/ting

["Ting" is a real word](https://www.merriam-webster.com/dictionary/ting), but even still I doubt it obviously comes to mind for most folks. I think it's generally more [*onomatopoeia*](https://www.merriam-webster.com/dictionary/onomatopoeia), in that it's mimicking its own "sound" rather than being a normal word you'd commonly use.

To account for adding the two parts of the name together, I "invented" (in the most honest way I knew how) to describe the intended syllable emphasis, by putting the emphasis mark on "ting", so that you pronounce it as "TING hooks" instead of as "ting HOOKS". But I didn't really have any more sophistication to it than that.

I've certainly seen examples of IPA before, and while I'm sure that it's more "universal" to do so, I don't find them particularly helpful, as an americanized-english speaker. My goal is for a lay person to be able to make sense of the phonetic pronunciation, not for a computer to be able to precisely parse it or something like that. If the name of some project is a real word with accepted pronunciation, I wouldn't need to put pronunciation in. But my projects are almost always invented from abbreviations or as [portmanteaus](https://www.merriam-webster.com/dictionary/portmanteau).

----

As a side note, I had contemplated/researched if there was some web service where you could construct audible phonetic combinations of sounds, like the little "play" buttons wikipedia and dictionaries often have, so people could just *hear* my intended pronunciation instead of needing to work it out visually.

I'm not aware of any such service, but I really wish it existed. I even thought about building it as a side project. But as I researched various differences in phonetic "standards", it seemed too hard for a weekend side project so I just dropped the idea.

----

These phonetic pronunciations are not intended to be super dictionary precise, but more like hints. That said, I do want to be helpful/correct rather than just making stuff up. I try to put them into my projects if there's any chance of ambiguity (there usually is), because I don't want people to make up their own imagined pronunciations if I can avoid it. Not that this would ever happen, but I don't want one of my projects to get so popular as GIF did and then have decades of endless debate as to the proper way to pronounce it. :)

**Comment by getify** [comment]
I found where I saw the "o͝o" at: https://en.wiktionary.org/wiki/hook

**Comment by JuanGaray93** [comment]
RE: the speech service, there are some online IPA parsers that translate text to speech but AFAIK most of them sound really... clunky. But I'm interested in this as well so if I ever see a decent one I'll try to send it your way!

I followed the links you provided and apparently the notation is enPR, also known as [American Heritage Dictionary](https://en.wikipedia.org/wiki/Phonetic_notation_of_the_American_Heritage_Dictionary) or AHD! I didn't recognise the notation as (I _think_) it's only used in America.

I really appreciate the time you took to reply to this, since I'm not a native English speaker I have some extra anxiety when meeting new English words. I regret having framed this issue as "this is confusing" now. *I* was confused. That's on me.

This said, would you mind if I created a PR adding the link to Wikipedia to the README just in case someone else finds it useful?

**Comment by getify** [comment]
Yeah that would be great and I appreciate your feedback and any help you can offer to improve!
